reviewdog.go

```diff
@@ -35,6 +40,8 @@ type CheckResult struct {
 	Lines   []string // Original error lines (often one line)
 }
 
+var vetError = fmt.Sprintf("", 1, 2, 3) // TODO: revert
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
var vetError is unused